### DISCUSSION TITLE
十一月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#638] [通知]
## 🚀 功能

- 支持点击评论/视频条目的用户头像显示用户信息 @Richasy (#637)
- 支持搜索建议 @Richasy (#635)

## 🐛 Bug 修复

- 修复PGC内容没有记录播放进度的问题 @Richasy (#636)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2111.2.0'>十一月更新 - 2</a>.</em>

**Comment by Ceacer** [comment]
老铁，能加个看直播和影视的历史嘛 ^^

**Comment by Dabaozimrz** [comment]
大佬浅色背景下文字也是显示白色，是我自己的问题吗

**Comment by AstbReal** [comment]
最新版本好像出现了播放视频卡顿的现象，我设置了N卡的控制面板，问题依旧，退回11月1次更新后正常。

**Comment by DDDandWWW** [comment]
提个小建议，看完视频觉得点返回按钮太麻烦了，我觉得可以把返回按钮设置成那种和“显示桌面”一样能盲操的那种。毕竟这个按钮太重要了，每次点击都需要把鼠标对准一个小按钮有点影响体验。或者也可以加入一个鼠标中键返回，干脆省的点了。
